static/css/impala/developers.less

```diff
@@ -2,25 +2,32 @@
 
 .developer-hub {
     #masthead .site-title {
-        small, strong {
+
+        small,
+        strong {
```

Comment:
Why do we have so much changes here?

Reply:
looks like me changing the file triggered a less reformatting setting/extension in VSC.  I can revert all the formatting changes?

Reply:
> I can revert all the formatting changes?

yes please, thanks!

Reply:
https://github.com/mozilla/addons-server/pull/16454/commits/e02affa0ab427cc83c8415c0e3fe6f2b91299845 now with even less 🟩